ui-v2/tests/acceptance/dc/services/show/services.feature

```diff
@@ -19,16 +19,22 @@ Feature: dc / services / gateway
     And I see linkedServices on the tabs
     When I click linkedServices on the tabs
     And I see linkedServicesIsSelected on the tabs
-  Scenario: Seeing the list of Linked Services
-    Given 3 service models from yaml
+  Scenario: Don't see the Linked Services tab
+    Given 1 datacenter model with the value "dc1"
+    And 1 node models
+    And 1 service model from yaml
+    ---
+    - Service:
+        Name: ingress-gateway-1
+        Kind: ingress-gateway
+    ---
     When I visit the service page for yaml
     ---
       dc: dc1
-      service: terminating-gateway-1
+      service: ingress-gateway-1
     ---
-    And the title should be "terminating-gateway-1 - Consul"
-    When I click linkedServices on the tabs
-    Then I see 3 service models
+    And the title should be "ingress-gateway-1 - Consul"
+    And I don't see linkedServices on the tabs
 
     
 

```

Comment:
Looks like the terminating-gateway tests were deleted here? Is that right?

Reply:
😬 added back in.